core/templates/dev/head/pages/base.html

```diff
@@ -1,5 +1,5 @@
 {% macro warnings_and_loader() -%}
-  <div ng-cloak>
+  <div id="blk-content" ng-cloak>
```

Comment:
Yes it was for blk. Something "oppia-main-content" does sound best to me if it is wrapping the main content. I wasn't entirely sure if it was appropriate to put a "main-content" id however, as it is only wrapping the alertsServices messages within the warnings_and_loader block rather than main content of the page right? 

I was under the impression a "main-content" wrapping tag should be literally be around the html, which produces the main content. So perhaps "oppia-block-content" would actually be better if you agree?